crypto/x509/standard_exts.h

```diff
@@ -71,7 +72,13 @@ static const X509V3_EXT_METHOD *standard_exts[] = {
     &ossl_v3_utf8_list[0],
     &ossl_v3_issuer_sign_tool,
     &ossl_v3_tls_feature,
-    &ossl_v3_ext_admission
+    &ossl_v3_ext_admission,
+    &ossl_v3_ext_admission,
```

Comment:
Why is ossl_v3_ext_admission duplicated?

Reply:
My mistake. I thought the diff was just the comma added at the end. Fixed.